### ISSUE TITLE
[PRE REVIEW]: Colette: A networking program for coffee-pairings to build research connections

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#3126]
**Submitting author:** @gacphillips (<a href="http://orcid.org/0000-0002-2347-2321">Genevieve A. C. Phillips</a>)
**Repository:** <a href="https://github.com/gacphillips/coffee-roulette" target ="_blank">https://github.com/gacphillips/coffee-roulette</a>
**Version:** v0.1.4
**Editor:** Pending
**Reviewer:** Pending
**Managing EiC:** Kevin M. Moerman

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**Author instructions**

Thanks for submitting your paper to JOSS @gacphillips. **Currently, there isn't an JOSS editor assigned** to your paper.

The author's suggestion for the handling editor is @timtroendle.

@gacphillips if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.5281/zenodo.3700700 is OK
- 10.5281/zenodo.4284845 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.02 s (236.0 files/s, 17582.0 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Markdown                         2             43              0            184
TeX                              1              1              0             47
YAML                             1              1              4             18
-------------------------------------------------------------------------------
SUM:                             4             45              4            249
-------------------------------------------------------------------------------


Statistical information for the repository '8c18fbfa8ac955cfa247f6c0' was
gathered on 2021/03/22.
No commited files with the specified extensions were found.
```

**Comment by whedon** [comment]
Failed to discover a valid open source license.

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03126/joss.03126/10.21105.joss.03126.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03126/joss.03126/10.21105.joss.03126.pdf) :page_facing_up: :point_left:

**Comment by Kevin-Mattheus-Moerman** [comment]
@gacphillips thanks for this submission. After reviewing the paper and the software (which I found to be here: https://github.com/gacphillips/colette rather than the above repository which does not contain software) we have determined this work is not in scope for JOSS. In particular it does not appear to have a significant research application. Furthermore the work is deemed too minor in terms of the amount of scholarly effort we look for. JOSS submissions under 1000 lines of code (LOC) are flagged for scope review and those under 300 LOC will be desk rejected (see also: https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort). Note that the fact that this work is not in scope for JOSS does not mean the work is of a poor quality or not useful. 

We will now proceed to reject this submission. We do hope you will consider JOSS for publication of more substantial software submissions with a clear scientific research application.
